shell/platform/common/cpp/client_wrapper/testing/stub_flutter_api.h

```diff
@@ -67,6 +67,19 @@ class StubFlutterApi {
   virtual void MessengerSetCallback(const char* channel,
                                     FlutterDesktopMessageCallback callback,
                                     void* user_data) {}
+
+  // Called for FlutterDesktopRegisterExternalTexture.
+  virtual int64_t RegisterExternalTexture(
```

Comment:
TextureRegistrarRegisterExternalTexture (see comment about the function namen in the C header). Same for the ones below.